.github/workflows/set-milestone-on-pr.yml

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.  and others.
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v. 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the Eclipse
+# Public License v. 2.0 are satisfied: GNU General Public License, version 2
+# with the GNU Classpath Exception which is available at
+# https://www.gnu.org/software/classpath/license.html.
+#
+# SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+# Set milestone on each pull request by using the next minor version
+# next version is computed using npm version tool
+on:
+  pull_request:
+    branches: [master]
+    types: [closed]
+
+jobs:
+  set-milestone:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - id: compute-milestone
+        run: |
+          cd packages/core
+          echo "MILESTONE_NUMBER=$(npm --no-git-tag-version version minor | cut -c 2-)" >> $GITHUB_ENV
```

Comment:
You could have read the version from `lerna.json` in the github script, but lgtm.

Reply:
yes what is better with package.json is that `npm version minor` is bumping the version automatically

so for example it will read 1.7.0 for the current state.

Reply:
Oh ok, so it voids my previous comment, thanks!